worlds/oot_soh/test/bases.py

```diff
@@ -1,5 +1,17 @@
 from test.bases import WorldTestBase
-
+from BaseClasses import Item, ItemClassification
+from ..Items import Items
 
 class SohTestBase(WorldTestBase):
     game = "Ship of Harkinian"
+    glitches_item_name = Items.GLITCHED
+
+    def enable_glitched_item(self):
+        """
+        Enable the use of the glitched/sequence breaking item for unit test purposes.
+
+        Also automatically award the item for convenience sake.
+        """
+        self.world.push_precollected(self.world.create_item(Items.GLITCHED))
+        self.multiworld.itempool.append(Item(Items.GLITCHED, ItemClassification.progression, None, 1))
+        self.collect_by_name(Items.GLITCHED)
```

Comment:
```suggestion
    def enable_glitched_item(self):
        """
        Enable the use of the glitched/sequence breaking item for unit test purposes.

        Also automatically award the item for convenience sake.
        """
        self.collect(self.world.create_item(Items.GLITCHED))
```
This will just collect it into state

Reply:
Applied. Good catch.